db/sql/index.js

```diff
@@ -41,6 +41,20 @@ module.exports = {
 	},
 	peers: {
 		// sql to be included
+	},
+	rounds: {
+		getVotes: load('rounds/getVotes.sql'),
+		updateVotes: load('rounds/updateVotes.sql'),
+		updateBlockId: load('rounds/updateBlockId.sql'),
+		summedRound: load('rounds/summedRound.sql'),
+		clearRoundSnapshot: load('rounds/clearRoundSnapshot.sql'),
+		performRoundSnapshot: load('rounds/performRoundSnapshot.sql'),
+		restoreRoundSnapshot: load('rounds/restoreRoundSnapshot.sql'),
+		clearVotesSnapshot: load('rounds/clearVotesSnapshot.sql'),
+		performVotesSnapshot: load('rounds/performVotesSnapshot.sql'),
+		restoreVotesSnapshot: load('rounds/restoreVotesSnapshot.sql'),
+		getMemRounds: load('rounds/getMemRounds.sql'),
+		flush: load('rounds/flush.sql'),
+		truncateBlocks: load('rounds/truncateBlocks.sql')
```

Comment:
@vitaly-t Everything looks good to me. Later we may need to introduce some auto loading pattern. So we just need to create an SQL file and just use it with its name in db repos. May be using `require-all` we can do it.

Reply:
NOOO 😸 That's completely in the wrong direction. Those are all dynamic file links, they are not just some SQL-loaded content. They are self-debugging, self-updating, self-error-reporting, etc. `require-all` would kill all that :smile: